Examples/src/main/kotlin/com/mapbox/vision/examples/POIActivityKt.kt

```diff
@@ -0,0 +1,262 @@
+import com.mapbox.mapboxsdk.geometry.LatLng
+import com.mapbox.vision.VisionReplayManager
+import com.mapbox.vision.examples.poi.POI
+import com.mapbox.vision.examples.poi.POIDrawData
+import com.mapbox.vision.examples.poi.POIState
+import com.mapbox.vision.mobile.core.interfaces.VisionEventsListener
+import com.mapbox.vision.mobile.core.models.AuthorizationStatus
+import com.mapbox.vision.mobile.core.models.Camera
+import com.mapbox.vision.mobile.core.models.Country
+import com.mapbox.vision.mobile.core.models.FrameSegmentation
+import com.mapbox.vision.mobile.core.models.classification.FrameSignClassifications
+import com.mapbox.vision.mobile.core.models.detection.FrameDetections
+import com.mapbox.vision.mobile.core.models.position.GeoCoordinate
+import com.mapbox.vision.mobile.core.models.position.GeoLocation
+import com.mapbox.vision.mobile.core.models.position.VehicleState
+import com.mapbox.vision.mobile.core.models.road.RoadDescription
+import com.mapbox.vision.mobile.core.models.world.WorldCoordinate
+import com.mapbox.vision.mobile.core.models.world.WorldDescription
+import kotlin.math.min
+import kotlinx.android.synthetic.main.activity_main.vision_view
+import kotlinx.android.synthetic.main.activity_poi.*
+
+class POIActivityKt : BaseActivity() {
+
+    companion object {
+
+        // POI will start to appear at this distance, starting with transparent and appearing gradually
+        private const val DRAW_LABEL_MIN_DISTANCE_METERS = 400
+
+        // POI will start to appear from transparent to non-transparent during this first meters of showing distance
+        private const val DISTANCE_FOR_ALPHA_APPEAR_METERS = 150
+        private const val LABEL_SIZE_METERS = 8
+        private const val LABEL_ABOVE_GROUND_METERS = 4
+    }
+
+    private var poiList = listOf<POI>()
+
+    private var visionReplayManagerWasInit = false
+
+    // Download session from tutorial and push to device
+    private var sessionPath = "${Environment.getExternalStorageDirectory().absolutePath}/session"
+
+    private val visionEventsListener = object : VisionEventsListener {
+
+        private var cameraCalibrated = false
+        private val paint = Paint()
+        private var bitmapCameraFrame = Bitmap.createBitmap(1, 1, Bitmap.Config.ARGB_8888)
+
+        override fun onAuthorizationStatusUpdated(authorizationStatus: AuthorizationStatus) {}
+
+        override fun onFrameSegmentationUpdated(frameSegmentation: FrameSegmentation) {}
+
+        override fun onFrameDetectionsUpdated(frameDetections: FrameDetections) {}
+
+        override fun onFrameSignClassificationsUpdated(frameSignClassifications: FrameSignClassifications) {}
+
+        override fun onRoadDescriptionUpdated(roadDescription: RoadDescription) {}
+
+        override fun onWorldDescriptionUpdated(worldDescription: WorldDescription) {}
+
+        override fun onCameraUpdated(camera: Camera) {
+            if (camera.calibrationProgress == 1.0f && !cameraCalibrated) {
```

Comment:
Talking strictly - it is theoretically possible `Camera.frameWidth` or `Camera.frameHeight` gets changed in runtime, I think it's worth to add that check here.

Reply:
Yeah guess for the simple example guess it's not worth to handle